common/src/main/java/com/wynntils/core/features/FeatureRegistry.java

```diff
@@ -123,7 +122,7 @@ public static void init() {
         // debug
         registerFeature(new ConnectionProgressFeature());
         registerFeature(new PacketDebuggerFeature());
-        registerFeature(new DummyOverlayFeature());
+        // registerFeature(new DummyOverlayFeature());
```

Comment:
Is this due to https://github.com/Wynntils/Artemis/issues/138?

Reply:
well no, since i was getting annoyed by it in dev environment - i'd rather not have my whole screen colored into different sections, even if i'm just testing stuff

Reply:
We need a way to disable debug features / explicitly enable them

Reply:
maybe DebugFeature should extend UserFeature?

Reply:
Or maybe "Debug" is more of an aspect of either User or System Features.

In any case, we should be able to disable features. I think it's coming up soon. :) Or, can't we do it already using commands?

Reply:
we can disable user features atm, but debug features don't have any framework for being toggled